packages/core/admin/admin/src/translations/fi.json

```
@@ -0,0 +1,594 @@
+{
+  "Analytics": "Tilastot",
+  "Auth.components.Oops.text": "Tilisi on jäädytetty.",
+  "Auth.components.Oops.text.admin": "Jos tämä on virhe, ota yhteyttä järjestelmänvalvojaan.",
+  "Auth.components.Oops.title": "Hups!",
+  "Auth.form.active.label": "Aktiivinen",
+  "Auth.form.button.forgot-password": "Lähetä uudelleen",
+  "Auth.form.button.go-home": "Palaa kotiin",
+  "Auth.form.button.login": "Kirjaudu sisään",
+  "Auth.form.button.login.providers.error": "Emme voi yhdistää sinua valitun palveluntarjoajan kautta",
+  "Auth.form.button.login.strapi": "Kirjaudu sisään Strapilla",
+  "Auth.form.button.password-recovery": "Salasanan palautus",
+  "Auth.form.button.register": "Valmis aloittamaan",
+  "Auth.form.confirmPassword.label": "Salasanan vahvistus",
+  "Auth.form.currentPassword.label": "Nykyinen salasana",
+  "Auth.form.email.label": "Sähköposti",
+  "Auth.form.email.placeholder": "kai@doe.com",
+  "Auth.form.error.blocked": "Järjestelmänvalvoja on estänyt tilisi.",
+  "Auth.form.error.code.provide": "Koodi on virheellinen.",
+  "Auth.form.error.confirmed": "Tilisi sähköpostiosoitetta ei ole vahvistettu.",
+  "Auth.form.error.email.invalid": "Tämä sähköposti on virheellinen.",
+  "Auth.form.error.email.provide": "Käyttäjänimesi puuttuu.",
+  "Auth.form.error.email.taken": "Tämä sähköposti on jo käytössä",
+  "Auth.form.error.invalid": "Käyttäjänimesi tai salasanasi on virheellinen.",
+  "Auth.form.error.params.provide": "Tiedot ovat virheellisiä.",
+  "Auth.form.error.password.format": "Salasanasi ei voi sisältää symbolia `$` kolmea kertaa.",
+  "Auth.form.error.password.local": "Tällä tilillä ei ole salasanaa.",
+  "Auth.form.error.password.matching": "Salasanat eivät ole identtisiä.",
+  "Auth.form.error.password.provide": "Salasanasi puuttuu.",
+  "Auth.form.error.ratelimit": "Liian monta yritystä, yritä uudelleen minuutin kuluttua.",
+  "Auth.form.error.user.not-exist": "Tätä sähköpostiosoitetta ei ole olemassa.",
+  "Auth.form.error.username.taken": "Tämä nimi on jo käytössä",
+  "Auth.form.firstname.label": "Etunimi",
+  "Auth.form.firstname.placeholder": "Kai",
+  "Auth.form.forgot-password.email.label": "Anna sähköpostiosoitteesi",
+  "Auth.form.forgot-password.email.label.success": "Sähköpostin lähetys onnistui seuraavaan osoitteeseen",
+  "Auth.form.lastname.label": "Sukunimi",
+  "Auth.form.lastname.placeholder": "Doe",
+  "Auth.form.password.hide-password": "Piilota salasana",
+  "Auth.form.password.hint": "Salasanan tulee sisältää vähintään 8 merkkiä, 1 iso kirjain, 1 pieni kirjain ja 1 numero.",
+  "Auth.form.password.show-password": "Näytä salasana",
+  "Auth.form.register.news.label": "Pidä minut ajan tasalla tulevista uusista ominaisuuksista ja parannuksista (tekemällä näin hyväksyt {terms} ja {policy}).",
```

Needs a little tweaking to sound more natural
```suggestion
  "Auth.form.register.news.label": "Pidä minut ajan tasalla tulevista uusista ominaisuuksista ja parannuksista (tekemällä näin hyväksyt seuraavat: {terms} ja {policy}).",
```